sdk/communication/communication-chat/src/generated/src/models/mappers.ts

```diff
@@ -42,11 +42,11 @@ export const ChatMessageReadReceipt: coreHttp.CompositeMapper = {
     name: "Composite",
     className: "ChatMessageReadReceipt",
     modelProperties: {
-      senderId: {
-        serializedName: "senderId",
-        required: true,
+      senderCommunicationIdentifier: {
+        serializedName: "senderCommunicationIdentifier",
```

Comment:
Are we working on shortening the names in the service preview-5 to comply with the naming guidelines?
https://skype.visualstudio.com/SPOOL/_wiki/wikis/SPOOL.wiki/16334/Identifiers-and-MRIs?anchor=naming

@juancamilor @amrelroumy-msft